### DISCUSSION TITLE
auto-importing from a custom folder doesn't work

### DISCUSSION THREAD

**Discussion by v-romanyuk** [nuxt/nuxt#17737] [Questions]
Hi guys! I'm trying to add auto-importing composables from a custom folder but it doesn't work. It still auto-imports from 'composables' folder only. Am I thinking the correct way, or it should be done somehow differently?
Here's my nuxt.config.ts
`export default defineNuxtConfig({
  autoImports: {
    dirs: [
      '~/composables',
      '~/store'
    ]
  }
})`

Nuxt version: 3.0.0-27491748.3186841

**Comment by psto** [comment]
I think Nuxt only scans files at the [top level](https://v3.nuxtjs.org/guide/directory-structure/composables#how-files-are-scanned) of the `composables/` directory.

**Reply by v-romanyuk** [reply]
Yeah I know. I was just curious why there’s a “autoImports” option in NuxtSchema and thought it’s customizable. But anyways, thank you!
